Resources/input/fsh/eabpkv-rechnungsworkflow.fsh

```diff
@@ -0,0 +1,38 @@
+Profile: EABPKVRechnungsworkflow
+Title: "EABPKV Rechnungsworkflow"
+Parent: Task
```

Comment:
Have you considered using the "Claim" Resource instead?

Reply:
This Task profile is meant for tracking the businessStatus of the workflow, for representing the state-machine in a Task profile seemed more appropriate. All information about the services itself will be covered in an Invoice resource. We don't intend to represent the individual services of the Practitioner itself yet in a Claim resource. Only information related to the payment will be available as structured content.